src/pom.js

```diff
@@ -912,6 +912,9 @@ foam.POM({
     { name: "foam/u2/wizard/debug/DebugWAO",                                              flags: "web" },
     { name: "foam/u2/wizard/debug/DebugContextIntercept",                                 flags: "web" },
     { name: "foam/u2/wizard/debug/DebugWizardletView",                                    flags: "web" },
+    { name: "foam/u2/wizard/debug/TestWizard",                                            flags: "web" },
```

Comment:
Wouldn't hurt to switch the flags: to "web&debug", would it?

Reply:
What turns on the debug flag? Is it always enabled in development?

Reply:
I like the 'debug' flag idea.